code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -238,7 +237,13 @@
 	pin = /obj/item/firing_pin
 	manufacturer = MANUFACTURER_INTEQ
 
-/obj/item/gun/ballistic/shotgun/bulldog/minutemen
+/obj/item/gun/ballistic/shotgun/bulldog/suns
+	name = "\improper Bulldog-C Shotgun"
+	desc = "A shorter variation of the Bulldog manufactured by Scarborough Arms. A shortened variant for SUNS who require more faster wielding Bulldogs personal defence."
```

Comment:
```suggestion
	desc = "A variation of the Bulldog manufactured by Scarborough Arms for SUNS. Its shorter barrel is intended to provide additional maneuverability in personal defense scenarios."
```
This is a pretty big change from the original wording, but I'm not sure how to make the original sentence structure work.